controllers/feeds/getFeedChat.js

```diff
@@ -60,7 +60,9 @@ const getDetail = (req, b, id) => {
         username: `Anonymous ${
           b.object?.anon_user_info_emoji_name ?? b?.anon_user_info_emoji_name
         }`,
+        emoji_name: b.object?.anon_user_info_emoji_name ?? b?.anon_user_info_emoji_name,
         emoji_code: b.object?.anon_user_info_emoji_code ?? b?.anon_user_info_emoji_code,
+        color_name: b.object?.anon_user_info_color_name ?? b?.anon_user_info_color_name,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `emoji_name` and `color_name` have been added correctly, using optional chaining and nullish coalescing operator to handle potential undefined values. However, it's important to ensure that the data being accessed here (`b.object?.anon_user_info_emoji_name`, `b.object?.anon_user_info_color_name`) is expected to be present in the `b` object and that these fields are not sensitive or private user information that should not be exposed.
```diff
+        emoji_name: b.object?.anon_user_info_emoji_name ?? b?.anon_user_info_emoji_name,
+        color_name: b.object?.anon_user_info_color_name ?? b?.anon_user_info_color_name,
```
If these fields are not always guaranteed to be present in the `b` object, you might want to add error handling or default values to prevent potential issues downstream where these values are used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->